### ISSUE TITLE
Setting value to a slider when page is reloaded

### ISSUE THREAD

**Issue by avesel** [egorkhmelev/jslider#25]
Hi!

I am trialing the use of the slider in Cognos 10.1 environment on a simple form with one entry field and one slider.
Note: This question is not related to Cognos, it is related to the slider behaviour.  
The only issue so far is how to safely set a slider value after a page refresh based on a persistent value (within a text box which maintains state).

a) When the page is initialized, the slider functions ok (loads default value from the HTML section).
    // commented out so it displays in github

b) For each page reload a java script tries to update the slider value 
             jQuery("#Slider1").slider("value", someValue);  
This action completely trashes the slider when it is called for the first time: the values are invalid, the slider displays the range 1 .. 10 and the value is set to 10.
c) if I now refresh the page again, the slider picks up correct values and works correctly. 

To sum up, I have found a way for the slider to provide the desired functionality, but am wondering why is crashes when value is set to it via this function for the first time: 
             jQuery("#Slider1").slider("value", someValue);   

I have also noticed that you need to set a value in the in order to correctly initialize the slider
<input id="Slider1" type="slider" name="price", value="50" /> // commented out so it displays in github

If I remove the value from the HTML element and set the value in the slider  init section, then the slider is not correctly initialized.

   

and code

```
jQuery("#Slider1").slider({
    :
    value: "50",
    :
```

Thanks in advance, Alex Vesel
